src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+
```

Comment:
Perhaps you can add in comments before the methods to provide a brief overview of the method for readers to understand it better!